pkg/apis/numaflow/v1alpha1/pipeline_types.go

```diff
@@ -270,8 +270,19 @@ type PipelineSpec struct {
 	Lifecycle Lifecycle `json:"lifecycle,omitempty" protobuf:"bytes,4,opt,name=lifecycle"`
 	// Limits define the limitations such as buffer read batch size for all the vertices of a pipleine, they could be overridden by each vertex's settings
 	// +kubebuilder:default={"readBatchSize": 100, "udfWorkers": 100, "bufferMaxLength": 10000, "bufferUsageLimit": 80}
-	// +optinal
+	// +optional
 	Limits *PipelineLimits `json:"limits,omitempty" protobuf:"bytes,5,opt,name=limits"`
+	// Watermark enables watermark progression across the entire pipeline.
```

Comment:
Add some comments that it should be not updated after pipeline is created.

Reply:
done